cmd/kubeapps-apis/plugins/resources/v1alpha1/common/plugin.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type ResourcesPluginConfig struct {
+	TrustedNamespaces TrustedNamespaces
+}
+
+type TrustedNamespaces struct {
+	HeaderName    string
+	HeaderPattern string
+}
+
+func NewDefaultPluginConfig() *ResourcesPluginConfig {
+	// If no config is provided, we default to the existing values for backwards compatibility.
+	return &ResourcesPluginConfig{}
+}
+
+// ParsePluginConfig parses the input plugin configuration json file and returns the configuration options.
+func ParsePluginConfig(pluginConfigPath string) (*ResourcesPluginConfig, error) {
+
+	// Resources plugin config defines the following struct and json config
+	type resourcesConfig struct {
+		Resources struct {
+			Packages struct {
+				V1alpha1 struct {
+					TrustedNamespaces struct {
+						HeaderName    string `json:"headerName"`
+						HeaderPattern string `json:"headerPattern"`
+					} `json:"trustedNamespaces"`
+				} `json:"v1alpha1"`
+			} `json:"packages"`
+		} `json:"resources"`
+	}
+	var config resourcesConfig
+
+	// #nosec G304
```

Comment:
Hey @castelblanque, just for the sake of learning, I've never seen this kind of comment/annotation before, and I guess it should be processed by some kind of security linter/checker. Could you tell me what's the tool in charge of this and where is its configuration?

Reply:
This is a `gosec` annotation we use for ignoring false positives and wontfixes. See https://github.com/securego/gosec#annotating-code